{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {
    "deletable": false,
    "editable": false,
    "nbgrader": {
     "checksum": "6ccb664ffd03d6f15da6fdc233097174",
     "grade": false,
     "grade_id": "cell-74a159a2616a05d3",
     "locked": true,
     "schema_version": 1,
     "solution": false
    }
   },
   "source": [
    "---\n",
    "# BIO458 Independent Project\n",
    "---\n",
    "\n",
    "\n",
    "The purpose of this project is to get you to think on your own about genomics research. The three goals are to:\n",
    "\n",
    "1) Come up with a biological or computational question\n",
    "\n",
    "2) Analyze appropriate data to address the data\n",
    "\n",
    "3) Interpret your results in terms of how they address the question\n",
    "\n",
    "The report itself should be written and analyzed within this folder. The idea of the project is for you to so a small analysis that takes you from raw data to a figure or two.\n",
    "\n",
    "\n",
    "---\n",
    "# Layout of the report\n",
    "---\n",
    "Your project report should be broken into 3 or 4 sections\n",
    "\n",
    "\n",
    "## Title\n",
    "- An informative title that gives the reader a sense of what they are about to see\n",
    "\n",
    "## Introduction: ~500 wds (20%)\n",
    "The introduction should motivate the reader as to why they question is worth answering. In general introductions start broadly, narrow down to the focus of the paper and finish with the question(s) being addressed. \n",
    "\n",
    "Your grade will depend on:\n",
    "- Do you provide sufficient background\n",
    "- Is the question motivated well\n",
    "- Is the question clearly stated \n",
    "- Quality of writing\n",
    "\n",
    "\n",
    "## Methods/Results (60%)\n",
    "Unlike a standard scientific paper I want you to have your methods and results embedded in the notebook. You can use a mixture of code and text (either code comments or markdown). Given the interactive nature of Jupyter Notebooks you can interleave your methods and results or provide one and then the other. **I do not expect a written methods and results section** separate from this section. \n",
    "\n",
    " - Document where and how you acquired the data\n",
    " - Document what the data are as they relate to your question\n",
    " - Include code and explanation for all analysis you conduct\n",
    "     - When you include code you should explain the purpose of each analysis\n",
    "     - You need to provide enough detail that you would reasonably expect another biologist to understand what you are doing\n",
    " - If you create tables or figures be sure to provide clear axis labels, legends and caption explaining what the figure is demonstrating. \n",
    " \n",
    "You will be graded on:\n",
    " - Clarity of documentation\n",
    " - Clarity and effectiveness of code\n",
    " - The quality of analysis - does the analysis address the question\n",
    " - Appropriateness of the data for the question\n",
    " - Quality of figures/table - do they demonstrate the main result clearly\n",
    " - Quality of writing\n",
    " \n",
    "## Discussion: ~500wds (20%)\n",
    "The discussion should be an interpretation of your results. Put another way, the discussion takes the objectve results and explains what they mean in terms of biology. \n",
    "\n",
    "- Do you address your question\n",
    "- Is your interpretation scientifically sound\n",
    "- Do you put your work in a broader context (have similar results been found in other studies?)\n",
    "- Quality of writing\n",
    "\n",
    "---\n",
    "# Finding data\n",
    "---\n",
    "\n",
    "There are many resources for downloading data. Here are three important ones\n",
    "1. Sequence Read Archive (NCBI)     \n",
    "The [NCBI website](https://www.ncbi.nlm.nih.gov/sra) is a massive store of next generation sequencee data. You can search particular kinds of data (whole genome, mRNA, ChIPseq) from specific organisms. In general you can download fastq reads from there onto your own computer or the lab server\n",
    "2. NCBI Genomes\n",
    "The [NCBI Genome site](https://www.ncbi.nlm.nih.gov/genome/) has most of the sequenced genome and their gene annotations available for download.\n",
    "3. UCSC Genome Browser\n",
    "The [UCSC Genome Browswer](https://genome.ucsc.edu/) is an online tool for viewing genomes and their associated genomic annotations (tracks). In addition to viewing the genomes of many many organisms they also host a download site that allows you to download the genomes, transcriptomes, gene annotations etc etc of many organisms. \n",
    "\n",
    "In reality there are many many websites that hose genomic data. A little bit of searching will often get you what you want. If you are having a hard time tracking down appropriate data for your project talk to Prof. Ness or the Teaching assistant during lab\n",
    "\n",
    "## Types of starting data\n",
    "Although I will not officially constrain you to particualr types of data I want the data to be genome scale. Here are the most common forms of data that you might start with. Keep in mind you might start with only one type of data or you might start with multiple. It will depend on your question.\n",
    "\n",
    "1. Fastq reads from a next generation sequencing (whole genome, RNAseq, CHiPseq, ddRAD etc, microbiome)\n",
    "2. Fasta files of complete genomes, transcriptomes, proteins\n",
    "3. VCF files - annotations of variation in a particualr genome \n",
    "4. BAM files - genome alignments usually of NGS data.\n",
    "\n",
    "---\n",
    "# Example Projects\n",
    "---\n",
    "1. Does gene density along the chromosome influence patterns of genetic diversity?\n",
    "\n",
    "2. Are the expression levels of nearby chromosomes correlated with one another\n",
    "\n",
    "3. Is the GC content of genes correlated with expression level\n",
    "\n",
    "4. Are families of gene duplicates clustered along the chromosome?\n",
    "\n",
    "5. *de novo* assembly of a micbrobial genome \n",
    "\n",
    "6. *de novo* asssmbly of a transcriptome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": false,
    "nbgrader": {
     "checksum": "ed70b23d8894b626b76bbfee2622f164",
     "grade": true,
     "grade_id": "cell-f8dc89ca390b8ce3",
     "locked": false,
     "points": 100,
     "schema_version": 1,
     "solution": true
    }
   },
   "outputs": [],
   "source": [
    "# YOUR CODE HERE\n",
    "raise NotImplementedError()"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
